### DISCUSSION TITLE
Light and Dark theme

### DISCUSSION THREAD

**Discussion by howhathe** [emilk/egui#1728] [Q&A] [answered]
**How to make a button to make it do dark or light theme?** (Depends on the current theme, if there is a dark - then when you click on the button is put white, if there is light then when you click on the button is put dark)
Or: **How to make 2 buttons: "Light" and "Dark"**, so that when you click on the grid, put a white theme. And when you click on the dark: dark. 

I'm already trying to make these buttons **30-40 minutes**, do not get it.
I use: [eframe-template](https://github.com/emilk/eframe_template)

**Comment by emilk** [comment, marked as answer]
https://docs.rs/egui/latest/egui/widgets/fn.global_dark_light_mode_switch.html or https://docs.rs/egui/latest/egui/widgets/fn.global_dark_light_mode_buttons.html

**Reply by howhathe** [reply]
Yo, thanks!
